.gitignore

```diff
@@ -62,3 +62,4 @@ __debug_*
 .claude/
 CLAUDE.md
 thoughts/
+thoughts
```

Comment:
I symlinked my `thoughts` directory across multiple worktrees, but it wasn't being gitignored, so I added this.